messaging/kafka-streams-reactive-messaging/src/test/java/io/quarkus/ts/messaging/kafka/reactive/streams/DisabledOnRHBQAarch64NativeConditions.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.ts.messaging.kafka.reactive.streams;
+
+import org.junit.jupiter.api.extension.ConditionEvaluationResult;
+import org.junit.jupiter.api.extension.ExecutionCondition;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+import io.quarkus.test.services.quarkus.model.QuarkusProperties;
+
+public class DisabledOnRHBQAarch64NativeConditions implements ExecutionCondition {
+
+    @Override
+    public ConditionEvaluationResult evaluateExecutionCondition(ExtensionContext extensionContext) {
+        boolean isRHBQ = QuarkusProperties.getVersion().contains("redhat");
+        boolean isNative = QuarkusProperties.isNativePackageType();
+        boolean isAarch64 = System.getProperty("ts.arm.missing.services.excludes").equals("true");
```

Comment:
this makes me nervous, javadoc says `
the string value of the system property, or null if there is no property with that key.` maybe you should make it `"true".equals(System...)` ? Well, if it is passing ok, but isn't it strange? might be missing something here.

Reply:
The profile sets it to true: https://github.com/quarkus-qe/quarkus-test-suite/blob/main/pom.xml#L825

Reply:
Oh, I see, your concern is the other way around! Good point.

Reply:
Updated. But lemme run CI anyway for this, this currently is not in that much hurry